contents/docs/experiments/adding-experiment-code.mdx

```diff
@@ -148,7 +148,10 @@ if variant == "variant-name" {
 
 For any server-side events that are also goal metrics for your experiment, you need to include a property `$feature/experiment_feature_flag_name: variant_name` when capturing those events. This ensures that the event is attributed to the correct experiment variant (e.g., test or control).
 
-This step is not required for events that are submitted via our client-side SDKs (e.g., JavaScript web, iOS, Android, React, React Native).
+> With the exception of the Go library, this step is not required if the server has local evaluation enabled and is capable of evaluating flags locally (i.e. no property filters). Under these conditions, flag information is automatically appended to every event sent to PostHog.
```

Comment:
Lets keep the callout for the mobile clients.

Also, do we need to update the feature flag docs too?

```suggestion
> This step is not required for events that are submitted via our client-side SDKs (e.g., JavaScript web, iOS, Android, React, React Native).

> For our backend SDKs, with the exception of the Go library, this step is not required if the server has [local evaluation](/docs/feature-flags/local-evaluation) enabled and is capable of evaluating flags locally (i.e. no property filters). In these cases, flag information is automatically appended to every event sent to PostHog.
```

Reply:
> Also, do we need to update the feature flag docs too?

I'm not sure - tagging events with flag values is more relevant for experiments than for "normal" feature flag use cases. @neilkakkar?